openapi.yaml

```diff
@@ -1,279 +1,864 @@
+          schema:
+            $ref: '#/components/schemas/ErrorResult'
+  schemas:
+    LocationResult:
+      properties:
+        links:
+          $ref: '#/components/schemas/PaginationLinks'
+        data:
+          $ref: '#/components/schemas/LocationResource'
+        included:
+          type: array
+          items:
+            $ref: '#/components/schemas/ServiceResource'
+        meta:
+          $ref: '#/components/schemas/Meta'
+    LocationSetResult:
+      properties:
+        links:
+          $ref: '#/components/schemas/PaginationLinks'
+        data:
+          type: array
+          items:
+            $ref: '#/components/schemas/LocationResource'
+        included:
+          type: array
+          items:
+            $ref: '#/components/schemas/ServiceIncluded'
+        meta:
+          $ref: '#/components/schemas/Meta'
+    ServiceResult:
+      properties:
+        links:
+          $ref: '#/components/schemas/PaginationLinks'
+        data:
+          $ref: '#/components/schemas/ServiceResource'
+        included:
+          type: array
+          items:
+            $ref: '#/components/schemas/LocationIncluded'
+        meta:
+          $ref: '#/components/schemas/Meta'
+    ServiceIncluded:
+      properties:
+        id:
+          type: string
+        type:
+          type: string
+          enum: [service]
+        attributes:
+          $ref: '#/components/schemas/ServicesAttributes'
+        links:
+          $ref: '#/components/schemas/SelfLink'
+    LocationIncluded:
+      properties:
+        id:
+          type: string
+          example: 5c4f1efcbaada0c63e
+          description: ID of a location object
+        type:
+          type: string
+          enum: [service]
+        attributes:
+          $ref: '#/components/schemas/LocationsAttributes'
+        links:
+          $ref: '#/components/schemas/SelfLink'
+    LocationResource:
+      properties:
+        id:
+          type: string
+          example: 5c4f1efcbaada0c63e
+          description: ID of a location object
+        type:
+          type: string
+          enum: [service]
+        attributes:
+          $ref: '#/components/schemas/LocationsAttributes'
+        links:
+          $ref: '#/components/schemas/SelfLink'
+        relationships:
+          $ref: '#/components/schemas/RelationshipsInLocations'
+    ServiceSetResult:
+      properties:
+        links:
+          $ref: '#/components/schemas/PaginationLinks'
+        data:
+          type: array
+          items:
+            $ref: '#/components/schemas/ServiceResource'
+        meta:
+          $ref: '#/components/schemas/Meta'
+        included:
+          type: array
+          items:
+            $ref: '#/components/schemas/LocationIncluded'
+    ServiceResource:
+      properties:
+        id:
+          type: string
+          example: 5c4f1efcbaada0c63e
+          description: ID of a service object
+        type:
+          type: string
+          enum: [service]
```

Comment:
`[service]` is used many times and I think it should be reusable. Example: https://github.com/osu-mist/persons-api/blob/v2/openapi.yaml#L703-L705

Same thing for `[location]`.